Library/Homebrew/extend/os/ENV/super.rb

```diff
@@ -0,0 +1,5 @@
+require "ENV/super"
```

Comment:
It looks like `extend/ENV/super.rb` lacks a trailing `require "extend/os/ENV/super"` that should be referencing this file.